.plop/templates/reactComponentStory.tsx.hbs

```diff
@@ -1,7 +1,8 @@
+import { storiesOf } from "@storybook/react";
 import React from "react";
 
 import { {{> p_componentName}} } from ".";
-import { createStory } from "../baseStory";
 
-createStory("{{> p_componentName}}")
-  .add("default", () => <{{> p_componentName}} />);
+storiesOf("@components/atoms/{{> p_componentName}}", module)
```

Comment:
You need to create component group partial:

in `index.js`:
```
  plop.setPartial("p_componentGroup", "{{componentGroup}}");
```

and then use it here as
```
"@components/{{> p_componentGroup}}/{> p_componentName}}"
```

Reply:
👍 done